doc/src/manual/style-guide.md

```diff
@@ -307,6 +307,39 @@ Calling [`eval`](@ref) inside a macro is a particularly dangerous warning sign;
 macro will only work when called at the top level. If such a macro is written as a function instead,
 it will naturally have access to the run-time values it needs.
 
+## Write macro expressions where `@` is unambiguous
```

Comment:
For users who it will be the first time for them using macros, it won't be clear what "scope" means. I think the `@` sign used there, and also having it in ticks (``), makes its clear on what is meant.